docs/howto/manage-web-app-charms/use-web-app-charm.rst

```diff
@@ -28,3 +28,65 @@ script will run on every unit. The script is assumed to be idempotent (in other
 can be run multiple times) and that it can be run on multiple units simultaneously
 without issue. Handling multiple migration scripts that run concurrently
 can be achieved by, for example, locking any tables during the migration.
+
+Use observability
+-----------------
+
+First, :ref:`integrate your web app with the Canonical Observability
+Stack <integrate_web_app_cos>`.
+
+Get Grafana login details
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+View the endpoints by running:
+
+.. code-block::
+
+    juju show-unit catalogue/0 | grep url
+    juju run grafana/leader get-admin-password
+
+These commands output URLs and the default admin password. Look for the URL
+with the Grafana postfix with the syntax
+``http://<IP_ADDRESS>/<JUJU_MODEL_NAME>-grafana``. Here, ``JUJU_MODEL_NAME``
+is the name of the Juju model on which you deployed your web app.
+
+View Grafana dashboard
+~~~~~~~~~~~~~~~~~~~~~~
+
+To view the dashboard, append the ``/dashboards``
+postfix to the URL and log in using the admin password to view the dashboards
+overview page.
+
+To view the specific dashboard for your web app, click :guilabel:`General` and
+then on :guilabel:`WebApp Operator`, where "WebApp" is a stand-in for the
+framework of your web app.
+
+For Flask and Django apps, the dashboard displays default graphs such as:
+
+* Requests: Number of requests over time.
+* Status code count: Number of requests broken by responses status code.
+* Requests per second: Number of requests per second over time.
+* 2XX Rate: Portion of responses that were successful (in the 200 range).
+* 3XX Rate: Portion of responses that were redirects (in the 300 range).
+* 4XX Rate: Portion of responses that were client errors (in the 400 range).
+* 5XX Rate: Portion of responses that were server errors (in the 500 range).
+* Request average duration: Average duration of the requests over time.
+* Request duration percentile: The 50th, 90th, and 99th percentile of all the
+  request duration lengths after sorting them from slowest to fastest. For
+  example, the 50th percentile represents the length of time (or less) that
+  50\% of the requests lasted.
```

Comment:
This feels like reference information that belongs in the COS docs. Do we need it here?